schema_editor/prototype/add_new_related_content_type.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
```

Comment:
Is this code related to the first Flask-based version? If so, I'm wondering whether it needs to stick around in the repo. Perhaps it would be better to have it on its own branch?

Reply:
I copied the `prototype` directory over [from DRIVER](https://github.com/WorldBank-Transport/DRIVER/tree/a33fdd503972dc0f5c87d52f3844d5a19d4dc89a/schema_editor/prototype) when I first got working on this branch, but after spending a lot of time with the schema editor app I don't believe `prototype` does anything. I'll go ahead and remove it.